forgot_password/handlers/util/email.py

```diff
@@ -25,14 +25,25 @@ def __init__(self, **smtp_params):
     def send_mail(self, sender, to, subject, text, html=None, reply_to=None):
         """
         Send email to user.
+
+        Arguments:
+        sender - (tuple) a tuple of the form ('Name', 'sender@example.com')
+        to - (string) - recipient address
+        subject - (str) The subject of the message
+        text - (tuple or None) The text version of the message
+        html - (tuple or None) The HTML version of the message
+        reply_to - (tuple) a tuple of the form ('Name', 'reply@example.com')
         """
         encoding = 'utf-8'
         text_args = (text, encoding)
         html_args = (html, encoding) if html else None
         headers = []
 
-        if reply_to:
-            headers.append(('Reply-To', reply_to))
+        if reply_to and reply_to[1]:
```

Comment:
I think you can simply pass `reply_to` to `format_addresses`. `format_addresses` support both single string (an email address) or a tuple (name-email pair), see https://github.com/aspineux/pyzmail/blob/e1a19edc86dbae59b67e83dd8a0ef5ee5f663dbb/pyzmail/pyzmail/generate.py#L93.

Reply:
This is different from sender because pyzmail takes the second element of `sender` tuple (i.e. `sender[1]`) as a value to the SMTP `MAIL FROM` address.